src/oneD/IonFlow.cpp

```diff
@@ -69,14 +68,9 @@ void IonFlow::updateTransport(double* x, size_t j0, size_t j1)
     for (size_t j = j0; j < j1; j++) {
         setGasAtMidpoint(x,j);
         m_trans->getMobilities(&m_mobility[j*m_nsp]);
-        if (m_overwrite_eTransport && (m_kElectron != npos)) {
-            if (m_import_electron_transport) {
-                m_mobility[m_kElectron+m_nsp*j] = m_elecMobility[j];
-                m_diff[m_kElectron+m_nsp*j] = m_elecDiffCoeff[j];
-            } else {
-                m_mobility[m_kElectron+m_nsp*j] = 0.4;
-                m_diff[m_kElectron+m_nsp*j] = 0.4*(Boltzmann * T(x,j)) / ElectronCharge;
-            }
+        if (m_import_electron_transport) {
+            m_mobility[m_kElectron+m_nsp*j] = m_elecMobility[j];
+            m_diff[m_kElectron+m_nsp*j] = m_elecDiffCoeff[j];
```

Comment:
For some cases, the mobility of electron could be different. This function is here because it is convenient for user to change the mobility and diffusion coefficient. I will fix the member's name.